cypress.json

```diff
@@ -1,4 +1,5 @@
 {
+  "video": false,
```

Comment:
good change!

Reply:
Videos could be great if we could access them after failed build. Something to consider later. Right now they slow build down a little.